app/templates/Thread/show.latte

```diff
@@ -68,6 +70,19 @@
 
     </div>
     <hr>
+    <div class="text-center">
+        {if $thread->event_id}
+            {foreach $otherEventThreads as $otherThread}
+                {var $title = html_entity_decode(strip_tags($otherThread->title))}
+                {if $otherThread->id == $thread->id}
+                    <strong>{$title}</strong>
+                {else}
+                    <a n:href="Thread:show $otherThread->id">{$title}</a>
+                {/if}
```

Comment:
Pojďme tohle celé obalit `<span class="ml10 mr10">` a `</span>`. To přidá `margin-left: 10px; margin-right: 10px;` a bude to trochu vizuálně oddělené.